src/renderers/shaders/ShaderChunk/dithering_pars_fragment.glsl

```diff
@@ -0,0 +1,14 @@
+// based on https://www.shadertoy.com/view/MslGR8
+vec3 dithering( vec3 color ) {
+	//Calculate grid position
+	float grid_position = fract( dot( gl_FragCoord.xy - vec2( 0.5, 0.5 ) , vec2( 1.0 / 16.0, 10.0 / 36.0 ) + 0.25 ) );
+
+	//Shift the individual colors differently, thus making it even harder to see the dithering pattern
+	vec3 dither_shift_RGB = vec3( 0.25 / 255.0, -0.25 / 255.0, 0.25 / 255.0 );
```

Comment:
You could use a circular offset pattern that offsets R,G,B differently based on the grid_position.  Might be more effective, but if this hides everything already I guess you are okay.

Something like
```
vec3 dither_shift_RGB = vec3( sin( grid_position), sin( grid_position + PI*0.666), sin( grid_position+PI*1.333 ) ) * ( 0.25 / 255.0 ).
```
Although I guess sin functions are costly. Maybe another way to do an even distribution via a linear approximation that still sums to zero.